frontend/app/components/card/search-result-entity/CardSearchResultEntityEvent.vue

```diff
@@ -62,13 +62,9 @@
 </template>
 
 <script setup lang="ts">
-import type { Event } from "~/types/events/event";
-
-import { useLinkURL } from "~/composables/useLinkURL";
-import { BASE_BACKEND_URL_NO_V1 } from "~/utils/baseURLs";
 
 const props = defineProps<{
-  event: Event;
+  event: Activity;
```

Comment:
Via our discussion, let's do `CommunityEvent` here so that it's related to what we're calling it in the backend and also adds more cotext so help the community understand via the necessary name change from`Event` because of the default [Event interface](https://developer.mozilla.org/en-US/docs/Web/API/Event).